docs/src/rust-client/counter_contract_tutorial.md

```diff
@@ -299,17 +330,19 @@ client
     .add_account(&counter_contract.clone(), Some(counter_seed), false)
     .await
     .unwrap();
+#     Ok(())
```

Comment:
Is this a leftover?

Reply:
to make this compile, the `main` function must return a `Result`.